components/compositing/compositor.rs

```diff
@@ -1656,14 +1659,13 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
         }
     }
 
-    /// Composite either to the screen or to a png image or both.
+    /// Composite to the given target if any, or the current target otherwise.
     /// Returns Ok if composition was performed or Err if it was not possible to composite
-    /// for some reason. If CompositeTarget is Window or Png no image data is returned;
-    /// in the latter case the image is written directly to a file. If CompositeTarget
-    /// is WindowAndPng Ok(Some(png::Image)) is returned.
+    /// for some reason. If read_back is true, reads back and returns the output frame as
```

Comment:
Could this explain more about what `read_back` means? I saw `CompositeTarget::SharedMemory` will return image value. Maybe we say which CompositeTarget will return the value?

Reply:
good catch, this became outdated in b422d3c9a07fcd43047ae06ad650f9d0951eb9ea

Reply:
done in 1f07d7bd76edf21bfb54842da7b96e14b6023e37